entry/package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Remove this file from the commit as its a generated file

Reply:
removed  package-lock.json